akka-typed/src/main/scala/akka/typed/javadsl/BehaviorBuilder.scala

```diff
@@ -0,0 +1,141 @@
+/**
+ * Copyright (C) 2009-2017 Lightbend Inc. <http://www.lightbend.com>
+ */
+
+package akka.typed.javadsl
+
+import scala.annotation.tailrec
+import akka.japi.pf.FI
```

Comment:
I think that's the wrong function types to use. Those are from 2014, most of them are not even partial functions but defined in `pf` hm...

I think these should use `import akka.japi.{ Pair, function }` instead